packages/@angular/cli/plugins/suppress-entry-chunks-webpack-plugin.ts

```diff
@@ -8,11 +8,19 @@ export class SuppressExtractedTextChunksWebpackPlugin {
     compiler.plugin('compilation', function (compilation: any) {
       // find which chunks have css only entry points
       const cssOnlyChunks: string[] = [];
-        const entryPoints = compilation.options.entry;
+      const entryPoints             = compilation.options.entry;
+      const regExp                  = new RegExp(
+                '(' +
+                  '\.(css|scss|sass|less|styl)|' +
+                  'node_modules[\/\\\\]webpack-dev-server[\/\\\\]client[\/\\\\]index\.js' +
+                      '(\\?https?\:\/\/.+)?|' +
+                  'webpack[\/\\\\]hot[\/\\\\]dev-server' +
+                ')$',
+                'i'
+            );
         // determine which entry points are composed entirely of css files
         for (let entryPoint of Object.keys(entryPoints)) {
-          if (entryPoints[entryPoint].every((el: string) =>
-            el.match(/\.(css|scss|sass|less|styl)$/))) {
```

Comment:
Is the large Regex actually necessary?  If the problem is the inclusion of a query string should something similar to the following be sufficient (even in the case of hot reloading)?
`/\.(css|scss|sass|less|styl)([\?#]\w*)?$/`